src/dmd/declaration.d

```diff
@@ -1087,6 +1089,24 @@ extern (C++) final class OverDeclaration : Declaration
     }
 }
 
+/** Tri-state bool.
+ *
+ * See_Also: https://en.wikipedia.org/wiki/Three-valued_logic
+ */
+enum trool : ubyte
```

Comment:
I would sugest using a more descriptive name. Also, the dmd norm seems to be that enum declarations start with a capital letter.

Reply:
Done.